core/src/test/java/org/infinispan/xsite/AbstractXSiteTest.java

```diff
@@ -267,11 +267,10 @@ protected EmbeddedCacheManager addClusterEnabledCacheManager(TransportFlags flag
          //get the transport here as clone.read below would inject the same transport reference into the clone
          // which we don't want
          Transport transport = clone.transport().getTransport();
-         Marshaller marshaller = clone.serialization().getMarshaller();
          GlobalConfiguration original = gcb.build();
          clone.read(original);
          clone.transport().transport(transport);
-         clone.serialization().marshaller(marshaller);
+         clone.serialization().addContextInitializer(TestDataSCI.INSTANCE);
```

Comment:
I remember why I removed this line: it doesn't belong here, because configuring the SCI should be part of the test setup, and this method just clones an existing configuration and replaces some instances that are not safe to reuse.

A better solution IMO would be to only put it where it's needed, in this case in `BaseSiteUnreachableTest.createSites()`

Reply:
Makes sense, updated